Packs/Reco/Integrations/Reco/Reco_test.py

```diff
@@ -212,3 +265,54 @@ def test_max_fetch():
     max_fetch = 200
     result = get_max_fetch(max_fetch)
     assert result == max_fetch
+
+
+def test_update_reco_incident_timeline(requests_mock, reco_client: RecoClient) -> None:
+    incident_id = uuid.uuid1()
+    requests_mock.put(f"{DUMMY_RECO_API_DNS_NAME}/incident-timeline/{str(incident_id)}", json={}, status_code=200)
+    reco_client.update_reco_incident_timeline(incident_id=incident_id, comment="test")
+
```

Comment:
Maybe we can add an assertion here in order to test that the command actually inserts the right data into the outputs - relevant to all unit tests added